debian/ceph-mgr.install

```diff
@@ -21,6 +21,7 @@ usr/share/ceph/mgr/prometheus
 usr/share/ceph/mgr/rbd_support
 usr/share/ceph/mgr/restful
 usr/share/ceph/mgr/selftest
+tusr/share/ceph/mgr/stats
```

Comment:
tusr?

Reply:
@gmayyyha 
Maybe the first vshankar's 12 commits are old one, I didn't sync them yet.
This PR just cares about the last 4 ones. It should be rebased later.

Thanks.